robot/impl/local_robot_test.go

```diff
@@ -1,4 +1,4 @@
-package robotimpl_test
+package robotimpl
```

Comment:
Seems fine; so `robot_reconfigure_test.go` and `local_robot_test.go` are now just separate files testing basically the same logic? With hypothetically more focus on the reconfiguration side of things for the former?

Reply:
I would argue they've always been testing very similar things, but yes. I'd like to at some point soon go into these tests and decouple them from the rest of the RDK as well and consolidate the tests somewhat (https://viam.atlassian.net/browse/RSDK-2670)